agent/CMakeLists.txt

```diff
@@ -72,7 +72,14 @@ if(ENABLE_NETWORKGATEWAY)
 endif()
 
 # Configure and install the general config file
-configure_file(softwarecontainer-config.in softwarecontainer-config @ONLY)
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/softwarecontainer-config.in
+               ${CMAKE_CURRENT_BINARY_DIR}/softwarecontainer-config
+               @ONLY)
+# Copy the configuration to the service test directory
```

Comment:
I think this comment should say why we do this, the actual configure/move of file is already visible in the actual code, but it's not easy to understand why.